### DISCUSSION TITLE
Manager Node unable to forward data to Search node

### DISCUSSION THREAD

**Discussion by f4ncyz4nz4** [Security-Onion-Solutions/securityonion#13761] [Unsupported Versions]
Good morning,

I have a distributed deployment composed by:
- Manager
- Search
- Sensor
- Receiver

My problem is that I correctly installed the Manager first and then the Search node, but I cannot see data in the Search, and if I try to use the command `cat /opt/so/log/elasticsearch/securityonion.log` I get the warning:
```
[2024-10-01T15:45:22,154][WARN ][org.elasticsearch.cluster.coordination.Coordinator] This node is a fully-formed single-node cluster with cluster UUID [xllk4sV4TEG40HXOhnyHnQ], but it is configured as if to discover other nodes and form a multi-node cluster via the [discovery.seed_hosts=[ip-10-10-10-237, search]] setting. Fully-formed clusters do not attempt to discover other nodes, and nodes with different cluster UUIDs cannot belong to the same cluster. The cluster UUID persists across restarts and can only be changed by deleting the contents of the node's data path(s). Remove the discovery configuration to suppress this message.
```
The things is I want the multi-cluster configuration but apparently after the installation the configuration never changed, but if I use the command `cat /opt/so/conf/elasticsearch/elasticsearch.yml` i get:
```
discovery:
  seed_hosts:
  - ip-10-10-10-237
  - search
```
So the manager cannot reach the Search but it's not a firewall problem.

Also it is not possible to join cluster manually because a configuration for `enrollment-token` is needed (as reported from [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/add-elasticsearch-nodes.html#:~:text=To%20add%20a%20node%20to,the%20HTTP%20layer%20to%200.0.) but it's not present in the Security Onion configuration.

The Manager roles are:
```
master
data
remote_cluster_client
transform
```
The Search roles are:
```
data
data_hot
ingest
transform
```

How can I create a multi-cluster configuration, with data present only on Search node?

Thanks a lot

**Comment by cm-ops** [comment]
What does sudo `so-elasticsearch-query .` return for both the manager and search node?

**Reply by f4ncyz4nz4** [reply]
Thanks for the answer.
From the manager:
```
{
  "name" : "ip-10-10-10-237",
  "cluster_name" : "securityonion",
  "cluster_uuid" : "_na_",
  "version" : {
    "number" : "8.14.3",
    "build_flavor" : "default",
    "build_type" : "docker",
    "build_hash" : "d55f984299e0e88dee72ebd8255f7ff130859ad0",
    "build_date" : "2024-07-07T22:04:49.882652950Z",
    "build_snapshot" : false,
    "lucene_version" : "9.10.0",
    "minimum_wire_compatibility_version" : "7.17.0",
    "minimum_index_compatibility_version" : "7.0.0"
  },
  "tagline" : "You Know, for Search"
}
```
From the search node:
```
{
  "name" : "search",
  "cluster_name" : "securityonion",
  "cluster_uuid" : "_na_",
  "version" : {
    "number" : "8.14.3",
    "build_flavor" : "default",
    "build_type" : "docker",
    "build_hash" : "d55f984299e0e88dee72ebd8255f7ff130859ad0",
    "build_date" : "2024-07-07T22:04:49.882652950Z",
    "build_snapshot" : false,
    "lucene_version" : "9.10.0",
    "minimum_wire_compatibility_version" : "7.17.0",
    "minimum_index_compatibility_version" : "7.0.0"
  },
  "tagline" : "You Know, for Search"
}
```

**Reply by cm-ops** [reply]
Is that your manager's hostname? `"name" : "ip-10-10-10-237` It doesn't apper either node has a cluster UUID. What version are you on and was this a fresh install?

On your manager, do you see anything when you run `sudo so-elasticsearch-query _cluster/settings?pretty`
